build/Build.cs

```diff
@@ -294,9 +301,16 @@ void PublishPackage(CalamariPackageMetadata calamariPackageMetadata)
                 .SetRuntime(calamariPackageMetadata.Architecture)
                 .SetOutput(outputDirectory)
             );
-            SignDirectory(outputDirectory);
+            
+            if (!project.Name.Contains("Tests"))
```

Comment:
:+1:

nit:`EndsWith` would be more precise with our naming conventions, but I don't think it will matter too much.